__tests__/newsApp.test.js

```diff
@@ -86,6 +87,87 @@ describe("app", () => {
     });
   });
 
+  describe("/api/articles/:article_id/comments", () => {
+
+    test("201 POST: responds with the posted comment (Request body accepts an object with the following properties 'username', 'body')", () => {
+      const requestBody = {
+        'username': 'butter_bridge',
+        'body': 'Not sure I got the point'
+      };
+      return request(app)
+      .post("/api/articles/9/comments")
+      .send(requestBody)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        console.log(comment);
+        expect(comment).toEqual({
+          comment_id: 19,
+          body: 'Not sure I got the point',
+          article_id: 9,
+          author: 'butter_bridge',
+          votes: 0,
+          created_at: expect.any(String)
+        });
+      });
+    });
+
+    test("400 POST: responds with an error message if input is not an object", () => {
+      const requestBody = [
```

Comment:
I wouldn't test this explicitly as we should always expect the request body to be an object.